pkg/controller/persistentvolume/controller_base.go

```diff
@@ -139,14 +139,14 @@ func (ctrl *PersistentVolumeController) initializeCaches(volumeSource, claimSour
 		glog.Errorf("PersistentVolumeController can't initialize caches: %v", err)
 		return
 	}
-	volumeList, ok := volumeListObj.(*api.List)
+	volumeList, ok := volumeListObj.(*api.PersistentVolumeList)
```

Comment:
See this comment: https://github.com/kubernetes/kubernetes/pull/28002/files#r68476481

I think your code change here is correct, but you need to fix the tests...